resources/views/components/developer-logins.blade.php

```diff
@@ -1,35 +1,33 @@
-@foreach ($users as $label => $credentials)
-    @if ($loop->first)
-        <div class="flex flex-col gap-y-6">
-            <div class="relative flex items-center justify-center text-center">
-                <div class="absolute border-t border-gray-200 w-full h-px"></div>
-                <p class="inline-block relative bg-white text-sm p-2 rounded-full font-medium text-gray-500 dark:bg-gray-800 dark:text-gray-100">
-                    {{ __('filament-developer-logins::auth.login-as') }}
+@if (! empty($users))
```

Comment:
I moved some logic outside of the loop (the "login as" and the errors placeholder). I've also changed the error text to `text-sm` to align with the rest of the error messages in the login form